src/nl/codefox/contextvh/tygronGoals.mod2g

```diff
@@ -4,6 +4,17 @@ order = linearall.
 module tygronGoals {
 	%_______________________________Adopt goals based on the environment_______________________________%
 	
+	%%% Selling buildings
+	
+	% If we want to sell a building but failed to do it before, we demolish it.
+	if bel(building(BuildingID,Name,OwnerID,_,Categories,FunctionID,_,MultiPoly,_,_),ourID(OwnerID),not(housingBuildings(Categories)),
+		failedToSell(MultiPoly))
```

Comment:
As this functionality has not yet been triggered, please check if this works correctly. Because this functionality does not break other code I agree to do so after the merge.